docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md

```diff
@@ -14,18 +14,24 @@ Enter the following when prompted:
     `/todo`
 ? Choose a Lambda source 
     `Create a new Lambda function`
-? Provide a friendly name for your resource to be used as a label for this category in the project: 
-    `todo`
-? Provide the AWS Lambda function name: 
+? Provide an AWS Lambda function name: 
     `todo`
-? Choose the function runtime that you want to use: 
+? Choose the runtime that you want to use: 
     `NodeJS`
 ? Choose the function template that you want to use: 
     `Serverless ExpressJS function (Integration with API Gateway)`
+? Do you want to configure advanced settings? 
+    `Yes`
 ? Do you want to access other resources created in this project from your Lambda function? 
```

Comment:
Just a few questions:

- Why do we want to configure advanced settings when adding a simple REST endpoint?
- Are the default values for the following questions same as when we select `no` for advanced settings?

Reply:
I answered `yes` to the prompt `Do you want to configure advanced settings?`, in order to see the next two prompts, which are currently shown in the tutorial. However, the default responses for the following prompts do appear to be `no`, so it does seem unnecessary to configure advanced settings. Since the two prompts `Do you want to access other resources created in this project from your Lambda function?` and `Do you want to invoke this function on a recurring schedule?` only appear when responding `yes` to configuring advanced settings, I'll remove those prompts.

Reply:
I answered `yes` to the prompt `Do you want to configure advanced settings?`, in order to see the next two prompts, which are currently shown in the tutorial. However, the default responses for the following prompts do appear to be `no`, so it does seem unnecessary to configure advanced settings. Since the two prompts `Do you want to access other resources created in this project from your Lambda function?` and `Do you want to invoke this function on a recurring schedule?` only appear when responding `yes` to configuring advanced settings, I'll remove those prompts.